packages/manager/src/managers/project/ProjectManager.ts

```diff
@@ -37,6 +38,16 @@ export class ProjectManager extends BaseManager {
 	private _cachedSliceMachineConfigPath: string | undefined;
 	private _cachedSliceMachineConfig: SliceMachineConfig | undefined;
 
+	async getSliceMachineDeprecatedConfigPath(): Promise<string | null> {
+		try {
+			return await locateFileUpward(SLICE_MACHINE_DEPRECATED_CONFIG_FILENAME, {
+				startDir: this.cwd,
+			});
+		} catch {}
+
+		return null;
+	}
+
```

Comment:
This feels fragile/unstructured to start adding "legecy" code to the manager, should this be handled by migrations? (migrating the old config file)

I think outside of the MigrationManager, other Managers shouldn't deal with legacy/deprecated patterns